### ISSUE TITLE
Stuff that needs to be finished before we can create a new pre-prepared dataset

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/nowcasting_dataset#393]
Issues that probable need to be done before we can start training ML models "properly":
- [x] #390 
- [x] #373 
- [x] #371 
- [x] #370 
- [x] #344 
- [x] #313 
- [x] #205 (only exclude GSPs with literally zero installed PV)
- [x] #410
- [x] #231
- [x] #469
- [x] #471

And some issues that aren't absolutely _essential_ but are pretty important:
- [x] #156
- [x] #447
- [x] MAKE A FEW BATCHES!
- [x] #325 
- [x] #341 
- [x] MAKE SOME MORE BATCHES!
- [x] Put three A6000s back in.
- [ ] #407
- [x] #454 
- [x] https://github.com/openclimatefix/nowcasting_dataset/issues/463
- [x] #483 
- [x] #482
- [x] https://github.com/openclimatefix/nwp/issues/6
- [x] #484
- [x] #486
- [x] https://github.com/openclimatefix/nowcasting_dataset/issues/488
- [ ] Create more batches (possibly after doing #485)
- [x] https://github.com/openclimatefix/nowcasting_dataset/issues/494

Nice to have:
- [ ] #491
- [ ] #425
- [ ] #317 
- [ ] #227
- [ ] #272

**Comment by peterdudfield** [comment]
I like this summary issues you do - nice to get a good overview of things

**Comment by JackKelly** [comment]
Thanks!  (Although I was also wondering if this list of issues actually be done using a github "milestone"... but really not sure what's best!)

**Comment by peterdudfield** [comment]
Ive added - https://github.com/openclimatefix/nowcasting_dataset/issues/454 to this list

**Comment by peterdudfield** [comment]
ive added https://github.com/openclimatefix/nowcasting_dataset/issues/483

**Comment by peterdudfield** [comment]
I'd be tempted to say this is done? 

This milestone of getting a 'prepared ML dataset' seems complete. There are a few nice to haves left over.

**Comment by JackKelly** [comment]
I agree!  Great work everyone!  Let's mark this as "closed" now.  Woo!
